src/Raven.Server/Indexing/VoronIndexOutput.cs

```diff
@@ -55,7 +55,7 @@ private Stream InitFileStream(StorageEnvironmentOptions options)
                 if (options.Encryption.IsEnabled)
                     return new TempCryptoStream(_fileTempPath, ignoreSetLength: true);
 
-                return SafeFileStream.Create(_fileTempPath, FileMode.CreateNew, FileAccess.ReadWrite, FileShare.ReadWrite, 4096, FileOptions.DeleteOnClose);
+                return SafeFileStream.Create(_fileTempPath, FileMode.CreateNew, FileAccess.ReadWrite, FileShare.ReadWrite, Voron.Global.Constants.Size.Megabyte, FileOptions.DeleteOnClose);
```

Comment:
I think that this may be a bad idea, see:
https://github.com/dotnet/runtime/blob/master/src/libraries/System.Private.CoreLib/src/System/IO/FileStream.cs#L749

The issue is that we are creating a 1MB managed buffer on each such call.

Reply:
Relevant: https://ayende.com/blog/171875/production-postmortem-the-case-of-the-memory-eater-and-high-load

Reply:
Then we need to buffer them.